src/FileHandler.cpp

```diff
@@ -0,0 +1,70 @@
+/**********************************************************************
+* OpenMC2 - An Open Source Re-Implementation of Midnight Club 2
+* Copyright (C) 2016  LRFLEW
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**********************************************************************/
+#include <algorithm>
+#include <cstdarg>
+#include <cstdio>
+#include "Game.hpp"
+#include "FileHandler.hpp"
+#include "Logging.hpp"
+
+void FileHandler::sub_617FB0(){
```

Comment:
This looks an awful lot like a destructor to me. Is this in the class' vtable? It probably isn't a destructor itself (likely just a close function), but still worth noting for when we do make a destructor for the class.